src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './utils/variables';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: calc(
+    #{$size-of-cell} * #{$number-of-columns} + #{$gap} * 6 + #{$padding} * 2
+  );
```

Comment:
The calculation inside 'calc' should be simplified. If '$size-of-cell', '$number-of-columns', '$gap', and '$padding' are defined as SASS variables, there's no need to use interpolation here. You can write it as 'calc($size-of-cell * $number-of-columns + $gap * 6 + $padding * 2)'.